dashboard/containers/dispatcher/fontprojects.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
+import csv
+import urllib
+CACHE_CSV = "fontprojects.csv"
+
+def get_repos():
+  handle = open(CACHE_CSV)
```

Comment:
If I'm not mistaken, garbage collection does its work of closing the file upon returning the function call, as there's no remaining reference to the file object.